GoogleUtilities/Reachability/GULReachabilityChecker.m

```diff
@@ -14,12 +14,17 @@
 
 #import <Foundation/Foundation.h>
 
-#import "GULReachabilityChecker+Internal.h"
-#import "Private/GULReachabilityChecker.h"
-#import "Private/GULReachabilityMessageCode.h"
-
+#import "Reachability/GULReachabilityChecker+Internal.h"
```

Comment:
What's the reason for these changing to have the root path?

Reply:
Partly to minimize the number of settings in the Package.swift and partly to experiment. 

There will need to be a thorough review and possible refactor of all imports before merging to master.